src/test_l1_dsDisplay.c

```diff
@@ -489,37 +473,37 @@ void test_l1_dsDisplay_positive_dsGetEDID(void) {
     result = dsDisplayInit();
     UT_LOG("\n In %s Return value: [%d]\n", __FUNCTION__, result);
     UT_ASSERT_EQUAL(result, dsERR_NONE);
-
+    
```

Comment:
cosmetic: remove space, check other places as well (trailing spaces). It shows as red bars when we use git. Also use 4 spaces instead of tab

Reply:
done